### PR TITLE
Add logs/run/rmi/rm integration test

### PR DESCRIPTION
**Pull request by jimmyxian**
Add logs/run/rmi/rm integration test
Signed-off-by: Xian Chaobo <xianchaobo@huawei.com>

### CODE REVIEW

File: test/integration/api.bats [hunk 1]
```diff
@@ -111,9 +111,24 @@ function teardown() {
...
+	# make sure container exists
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" ==  *"test_container"* ]]
+
+	# verify
+	run docker_swarm logs test_container
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 1 ]
+	[[ "${lines[0]}" ==  *"hello world"* ]]
```
Comment (abronan) [inline test/integration/api.bats]: We could probably add a second part here to test `logs` with multiple lines output.

**Comment by abronan** [comment]
Tests OK and overall LGTM. Just a minor comment on the log but not sure if really necessary.

/cc @aluzzardi

**Comment by aluzzardi** [comment]
Related to #492

File: test/integration/api.bats [hunk 2]
```diff
@@ -182,19 +197,101 @@ function teardown() {
...
+	run docker_swarm run -d --name test_container busybox sleep 500
+	[ "$status" -eq 0 ]
+
+	# make sure container exsists and is up
+	run docker_swarm ps -a
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" == *"test_container"* ]]
+	[[ "${lines[1]}" == *"Up"* ]]
+
+	# rm -f, remove a running container
```
Comment (aluzzardi) [inline test/integration/api.bats]: You could probably do `rm` before (with no `-f`) and make sure it fails

File: test/integration/api.bats [hunk 3]
```diff
@@ -182,19 +197,101 @@ function teardown() {
...
+
+	# verify
+	run docker_swarm ps -aq
+	[ "${#lines[@]}" -eq 0 ]
 }
 
-# FIXME
 @test "docker rmi" {
-	skip
+	start_docker 3
```
Comment (aluzzardi) [inline test/integration/api.bats]: LGTM

File: test/integration/api.bats [hunk 4]
```diff
@@ -182,19 +197,101 @@ function teardown() {
...
+		run docker -H $host images -q
+		[ "$status" -eq 0 ]
+		[ "${#lines[@]}" -eq 0 ]
+	done
 }
 
-# FIXME
 @test "docker run" {
-	skip
+	start_docker 3
```
Comment (aluzzardi) [inline test/integration/api.bats]: LGTM

File: test/integration/api.bats (hunk 2 above)
Comment (jimmyxian) [inline test/integration/api.bats]: OK

**Comment by jimmyxian** [comment]
@abronan
Aleady modified. I have added 3 lines to check

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**